examples/vanilla/2d_clock/package.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "2d_clock",
+  "private": true,
+  "version": "0.0.1",
+  "type": "module",
+  "scripts": {
+    "dev": "vite",
+    "build": "vite build",
+    "preview": "vite preview"
+  },
+  "devDependencies": {
+    "vite": "^4.4.5"
+  },
+  "dependencies": {
+    "@react-three/fiber": "^8.15.11",
```

Comment:
Is `react-three/fiber` required here?

Reply:
Good one! I ran the script from the '/scripts/updateAll.sh' file that runs every `/examples/**` folder to add/update packages needed for each examples. Maybe I need to tweak the bash file to run specific package folder on each framework examples.

Reply:
got it